FirestoreDocument.js

```diff
@@ -66,40 +66,79 @@ function wrapValue_ (value) {
 
 function unwrapValue_ (value) {
   var type = Object.keys(value)[0]
+  value = value[type]
   switch (type) {
+    case 'referenceValue':
+      // TODO: Allow possibility to retrieve document from this reference
```

Comment:
I'd like to keep TODOs out of code that is published in the library. Could you file this as an issue instead and delete the TODO?